db/migrate/20160425161234_remove_deleted_tables_from_replication_settings.rb

```diff
@@ -0,0 +1,16 @@
+class RemoveDeletedTablesFromReplicationSettings < ActiveRecord::Migration[5.0]
+  class SettingsChange < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+    serialize :value
+  end
+
+  def up
+    deleted_tables = %w(miq_events miq_license_contents vim_performances)
```

Comment:
@Fryguy I updated this to add `miq_event_definitions` to the exclude_tables list. I verfied that the list was never updated when `miq_events` was renamed.